config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main-upgrades.yaml

```diff
@@ -35,6 +35,8 @@ periodics:
           value: "v1.9.3"
         - name: GINKGO_FOCUS
           value: "\\[K8s-Upgrade\\]"
+        - name: AZURE_WORKLOAD_ID
+          value: "cabf5f22-ec7e-4e84-9e35-c02e57ca555d" # not a secret
```

Comment:
@jsturtevant does it make sense to expose this value distinctly from the subscription ID? Are these uuids sufficiently unique across subs, or are these (as I suspect) namespaced by sub?

Reply:
Yea, I think we will want subid and tenant Id all grouped together

Reply:
Resolving this conversation as Tenant ID and Subs ID has been added in the preset.